task60_PageObject/src/test/java/tut/by/tests/AuthorizationTest.java

```diff
@@ -0,0 +1,56 @@
+package tut.by.tests;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import tut.by.pages.HomePage;
+import tut.by.pages.LoginPage;
+import static org.testng.Assert.assertEquals;
+
+public class AuthorizationTest {
+    private static final String url = "https://www.tut.by/";
+    private static final String username = "seleniumtests@tut.by";
+    private static final String password = "123456789zxcvbn";
+    private static final String usertext = "Selenium Test";
+    private WebDriver driver;
+
+
+    @BeforeMethod
+    public void setUp(){
+        driver = new ChromeDriver();
+        driver.get(url);
+        driver.manage().window().maximize();
```

Comment:
Let's create Singleton for WebDriver. It will grant access to WebDriver from any place in the code + will help to avoid passing WebDriver instance via ctor parameter in page objects. Also you will need it in task with reporting + final task

Reply:
done